server/deskroom/knowledge_base/service.py

```diff
@@ -25,10 +39,66 @@ async def create_knowledge_base_create_job(
 
 
 async def mark_create_job_done(
-    supabase: AsyncClient, job_id: int
+    supabase: AsyncClient, job_id: str
 ) -> KnowledgeBaseCreateJob:
     response = await (
-        supabase.table("jobs").update({"status": 1}).eq("id", job_id).execute()
+        supabase.table("uploads")
+        .update({"status": "DONE"})
+        .eq("job_id", job_id)
+        .execute()
     )
 
     return KnowledgeBaseCreateJob.model_validate(response.data[0])
+
+
+def read_xlsx_from_azure_blob_storage(
+    blob_storage_url: str, client: ContainerClient
+) -> pd.DataFrame:
+    obj = urlparse(blob_storage_url)
+    [container, *blob_names] = obj.path[1:].split("/")
+    logger.info(f"{container=}, {blob_names=}")
+
+    blob_name = "/".join(blob_names)
+    logger.info(blob_name)
+
+    if not blob_name.endswith(".xlsx"):
+        raise ValueError("Invalid file format. Please upload an xlsx file.")
+    blob_client = client.get_blob_client(blob_name)
+    download = blob_client.download_blob()
+    return pd.read_excel(download.readall(), engine="openpyxl")
```

Comment:
채널톡에서 넘어오는 엑셀파일 내부에 시트가 많아서, Message data 시트를 명시해주어야 chatId 컬럼을 참조할 수 있습니다!

```suggestion
    return pd.read_excel(download.readall(), engine="openpyxl", sheet_name="Message data")
```

Reply:
요거 나중에도 바뀔수 있는 내용일까요? Sheet name이 고정일까요? 변수 혹은 함수 parameter로 받아야하는 지 궁금합니다.

Reply:
@shawnhyeonsoo 요거 함수 자체가 azure에서 읽어서 dataframe으로 읽어오는 거라서, 좀 더 범용적으로 쓰기 위해 sheet name을 인자로 추가했습니다.

Reply:
요거 아래 커밋에서 수정했는데 한번 리뷰 다시부탁드려요~ @shawnhyeonsoo